src/styles/layout/modal.less

```diff
@@ -31,6 +31,22 @@
     width: @modal-width-narrow;
   }
 
+  .icon-button-collection {
+
+    button {
+      height: auto;
+
+      &:last-child {
+        margin-bottom: 0;
```

Comment:
That's true, but might we use `icon-button-collection` without `button-collection-stacked` in the future if we introduce non-stacked buttons to the modal?